Mastodon/Scene/HomeTimeline/HomeTimelineViewModel.swift

```diff
@@ -163,12 +163,11 @@ extension HomeTimelineViewModel {
         case .home:
             response = try? await context.apiService.homeTimeline(
                maxID: status.id,
-               limit: 20,
                authenticationBox: authContext.mastodonAuthenticationBox
            )
         case .public:
             response = try? await context.apiService.publicTimeline(
-                query: .init(maxID: status.id, limit: 20),
+                query: .init(maxID: status.id),
```

Comment:
```suggestion
                query: .init(maxID: status.id),
                query: .init(local: true, maxID: status.id),
```

Just saw posts from other instances appearing in my local timeline 🫠